conf/messages

```diff
@@ -2896,66 +2896,88 @@ inYearLosses.agent.title=Do you want to include any other Capital Gains Tax loss
 inYearLosses.personalRep.helpText=For example, if a property was sold for less than it was bought for, the seller would make a loss.<br/> Including any other Capital Gains Tax losses the person made in this tax year will reduce their gain. The person must have made the losses before {0}.
+inYearLosses.personalRepInPeriodOfAdmin.helpText=An example of a loss would be where you, or the estate, sold a property or shares for less than the market value on the person’s date of death.<br/>Including any other Capital Gains Tax losses the estate made in this tax year will reduce its gain.
+inYearLosses.personalRepInPeriodOfAdmin.agent.helpText=	An example of a loss would be where your client, or the estate, sold a property or shares for less than the market value on the person’s date of death.<br/>Including any other Capital Gains Tax losses the estate made in this tax year will reduce its gain.
 
 inYearLosses.error.required=Select yes if you want to include any Capital Gains Tax losses you made in this tax year
 inYearLosses.agent.error.required=Select yes if you want to include any other Capital Gains Tax losses your client made in the tax year
 inYearLosses.trust.error.required=Select yes if you want to include any other Capital Gains Tax losses the trust made in the tax year
 inYearLosses.capacitor.error.required=Select yes if the person wants to include any Capital Gains Tax losses they made in this tax year
 inYearLosses.personalRep.error.required=Select yes if you want to include any Capital Gains Tax losses the person made in this tax year
+inYearLosses.personalRepInPeriodOfAdmin.error.required=Select yes if you want to include any Capital Gains Tax losses the estate made in this tax year
+inYearLosses.personalRepInPeriodOfAdmin.agent.error.required=	Select yes if your client wants to include any Capital Gains Tax losses the estate made in this tax year
 
 inYearLosses.error.invalid=Select yes if you want to include any Capital Gains Tax losses you made in this tax year
 inYearLosses.agent.error.invalid=Select yes if you want to include any other Capital Gains Tax losses your client made in the tax year
 inYearLosses.trust.error.invalid=Select yes if you want to include any other Capital Gains Tax losses the trust made in the tax year
 inYearLosses.capacitor.error.invalid=Select yes if the person want to include any Capital Gains Tax losses you made in this tax year
 inYearLosses.personalRep.error.invalid=Select yes if the person want to include any Capital Gains Tax losses you made in this tax year
+inYearLosses.personalRepInPeriodOfAdmin.error.invalid=Select yes if the person want to include any Capital Gains Tax losses you made in this tax year
+inYearLosses.personalRepInPeriodOfAdmin.agent.error.invalid=Select yes if the person want to include any Capital Gains Tax losses you made in this tax year
```

Comment:
this should be `your client` instead of `the person`

Reply:
fixed